### DISCUSSION TITLE
query-string boolean validation

### DISCUSSION THREAD

**Discussion by Legion112** [laravel/framework#32986] [General]
- Laravel Version: 6.16.0
- PHP Version: 7.4
- Database Driver & Version: does not matter 

### Description:
I have a queryString filter parameter `filters[is_active]`
I wanna define validation `'filters.is_active'  => ['boolean']`

But it does not work as expected when I pass parameter `false` it says `The filters.is active field must be true or false.`

Well I guess it does not work with `query-string` only with `POST` request ((|

I think we should change this behavior

### Steps To Reproduce:
Create a route and define validation there
```
$this->validate($request, [
                'filters.is_active' => [
                    'nullable',
                    'boolean',
                ],
])
```
Send queryString filters[is_active]=false

**Comment by driesvints** [comment]
converted to discussion.

**Reply by Legion112** [reply]
@driesvints  could you share the location where 'boolean' validation implementation is
I would like to take a look.
